x/virtualgroup/keeper/keeper.go

```diff
@@ -358,6 +365,9 @@ func (k Keeper) SwapAsPrimarySP(ctx sdk.Context, primarySP, successorSP *sptypes
 	}
 	k.SetGVGStatisticsWithSP(ctx, srcStat)
 	k.SetGVGStatisticsWithSP(ctx, dstStat)
+	k.DeleteGVGFamilyWithinSP(ctx, srcVGFStat.SpId, family.Id)
```

Comment:
if this is executed before the hardfork, there is no stats yet.

Reply:
fixed